lesson2/src/linked_list/queue.js

```diff
@@ -6,18 +6,30 @@ let Stack = require("../../../lesson1/src/stacks_queues/stack");
 class StackQueue {
   constructor() {
     // create 2 stacks here... you are not allowed to use an array or list
+    this.stack1 = [];
```

Comment:
I put a question on these code, but i have used an array to implement the stack.